src/containers/EditSubjectFrontpage/components/SubjectFrontpageForm.tsx

```diff
@@ -0,0 +1,223 @@
+  const [unsaved, setUnsaved] = useState(false);
+  usePreventWindowUnload(unsaved); //vil jeg ha denne?
+
+  const panels = [
+    {
+      id: 'subject-frontpage-about',
+      title: 'subjectFrontpageForm.about',
+      className: 'u-4/6@desktop u-push-1/6@desktop',
+      errorFields: ['title', 'description', 'visualElement'],
+      component: ({ t }: ComponentProps) => <SubjectFrontpageAbout />,
+    },
+    {
+      id: 'subject-frontpage-metadata',
+      title: 'subjectFrontpageForm.metadata',
+      className: 'u-6/6',
+      errorFields: ['metaDescription', 'banner'],
+      component: ({ t, hasError, closePanel }: ComponentProps) => (
+        <SubjectFrontpageMetadata t={t} />
+      ),
+    },
+    {
+      id: 'subject-frontpage-articles',
+      title: 'subjectFrontpageForm.articles',
+      className: 'u-6/6',
+      errorFields: ['editorChoices', 'mostRead'],
+      component: ({ editorsChoices }: ComponentProps) => (
+        <SubjectFrontpageArticles articles={editorsChoices} />
+      ),
+    },
+  ];
+
+  const getInitialValues = (subject: SubjectType) => {
+    const visualElementId = subject.about.visualElement.url.split('/').pop();
+    return {
+      articleType: 'subject-frontpage',
+      supportedLanguages: ['nb', 'nn'],
+      language: selectedLanguage,
+      description: subject.about.description,
+      aboutTitle: subject.about.title,
+      visualElement: {
+        resource: subject.about.visualElement.type,
+        url: subject.about.visualElement.url,
+        alt: subject.about.visualElement.alt,
+        resource_id: visualElementId,
+      },
+      visualElementAlt: subject.about.visualElement.alt,
+      visualElementCaption: subject.about.visualElement.caption,
+      image: subject.banner.desktopId,
+      bannerUrl: {
+        desktopUrl: subject.banner.desktopUrl,
+        mobileUrl: subject.banner.mobileUrl,
+      },
+      desktopBannerId: subject.banner.desktopId,
+      mobileBannerId: subject.banner.mobileId,
+      editorsChoices: subject.editorsChoices,
+      facebook: subject.facebook,
+      filters: subject.filters,
+      goTo: subject.goTo,
+      id: subject.id,
+      latestContent: subject.latestContent,
+      layout: subject.layout,
+      metaDescription: subject.metaDescription,
+      mostRead: subject.mostRead,
+      title: subject.name,
+      topical: subject.topical,
+      twitter: subject.twitter,
+      subjectId: subjectId,
+    };
+  };
+
+  const initialValues = getInitialValues(subject);
+  const editorsChoices = initialValues.editorsChoices;
+
+  return (
+    <Formik
+      enableReinitialize
+      initialValues={initialValues}
+      onSubmit={() => {}}>
+      {({ values, dirty, isSubmitting, setValues, errors, touched }) => {
+        const formIsDirty = isFormikFormDirty({
+          values,
+          initialValues,
+          dirty,
+        });
+
+        setUnsaved(formIsDirty);
+
+        var savedToServer = true;
+        const onSaveClick = () => {
```

Comment:
Lagring av forside må lage ny dersom faget ikkje har verdi i contentUri. Etter at forsida er oppretta må contentUri for subject i taksonomi oppdateres, så det må finnes et endepunkt for det i taksonomiApi.